### PR TITLE
Managers can edit/delete holidays from holiday index

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: app/assets/javascripts/calendars/holidays.es6 [hunk 1]
```diff
@@ -24,27 +24,10 @@
...
-                this.$el.fullCalendar('removeEvents', event._id);
-              }
-          });
-        }
-      });
-
-      element.append($button);
+    eventClick(event) {
+      if (!event.bank_holiday) {
+        window.location = 'holidays/'+event.holiday_ids+'/edit';
```
Comment (benlovell) [inline app/assets/javascripts/calendars/holidays.es6]: Can this be interpolated? It's a bit neater.

File: app/serializers/holiday_serializer.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 class HolidaySerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
```
Comment (benlovell) [inline app/serializers/holiday_serializer.rb]: This isn't needed?

File: app/views/holidays/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<div class="page-header">
+  <ol class="breadcrumb">
+    <li><a href="<%= root_path %>">Home</a></li>
+    <li><a href="<%= holidays_path %>">Holidays & unavailability</a></li>
```
Comment (Guntrisoft) [inline app/views/holidays/edit.html.erb]: Can you HTML escape this &amp;

File: app/assets/javascripts/calendars/holidays.es6 (hunk 1 above)
Comment (AndrewVos) [inline app/assets/javascripts/calendars/holidays.es6]: k done

File: app/views/holidays/edit.html.erb (hunk 3 above)
Comment (AndrewVos) [inline app/views/holidays/edit.html.erb]: done

File: app/assets/javascripts/calendars/holidays.es6 (hunk 1 above)
Comment (AndrewVos) [inline app/assets/javascripts/calendars/holidays.es6]: done

Comment (AndrewVos) [inline app/assets/javascripts/calendars/holidays.es6]: done

File: app/serializers/holiday_serializer.rb (hunk 2 above)
Comment (AndrewVos) [inline app/serializers/holiday_serializer.rb]: done

File: app/views/holidays/edit.html.erb (hunk 3 above)
Comment (AndrewVos) [inline app/views/holidays/edit.html.erb]: done

File: app/assets/javascripts/calendars/holidays.es6 [hunk 4]
```diff
@@ -24,27 +24,10 @@
...
-              complete: () => {
-                this.$el.fullCalendar('removeEvents', event._id);
-              }
-          });
-        }
-      });
-
-      element.append($button);
+    eventClick(event) {
+      if (!event.bank_holiday) {
```
Comment (Guntrisoft) [inline app/assets/javascripts/calendars/holidays.es6]: Can we not create a `url` attribute in the holidays feed? Then this method can be removed

Comment (AndrewVos) [inline app/assets/javascripts/calendars/holidays.es6]: ok done

**Comment by AndrewVos** [comment]
All comments replied to @benbarnett @benlovell @Guntrisoft

**Comment by AndrewVos** [comment]
@Guntrisoft 

https://github.com/guidance-guarantee-programme/telephone_appointment_planner/pull/185/commits/be80a1d2a4240a8f4bc18b570f6ae862529fd921

**Pull request merged**